vpr/SRC/base/vpr_types.h

```diff
@@ -724,6 +724,9 @@ struct s_placer_opts {
  * bend_cost:  Cost of a bend (usually non-zero only for global routing).   *
  * max_router_iterations:  Maximum number of iterations before giving       *
  *                up.                                                       *
+ * min_incremental_reroute_fanout: Minimum fanout a net needs to have 		*

```

Comment:
Consider using spaces instead of tabs for comments like this -- formatting breaks if tab size != 4